prac_07/myguitars.py

```diff
@@ -0,0 +1,43 @@
+"""
+My Guitars
+Guitar Program from Practical 7 Exercise
+"""
+from prac_07.guitar import Guitar
+
+FILENAME = "guitars.csv"
+NAME_INDEX = 0
+YEAR_INDEX = 1
+COST_INDEX = 2
+
+
+def main():
+    """Read guitars from file, store as list of Guitar objects and display sorted."""
+    with open(FILENAME, 'r') as in_file:
+        guitars = []
+        for line in in_file:
+            parts = line.strip().split(',')
+
+            name = parts[NAME_INDEX]
+            year = parts[YEAR_INDEX]
+            cost = parts[COST_INDEX]
+            guitar = Guitar(name, int(year), float(cost))
+            guitars.append(guitar)
```

Comment:
Is it necessary to have constants for the indexes ? 
Also might it be better to create the list outside of the file being open ?
Also is it necessary to create a variable to append a new guitar object ?
Here is an idea for how you could make the code simpler: 
```
    guitars = []
    with open(FILENAME, 'r') as in_file:
        for line in in_file:
            parts = line.strip().split(',')
            guitars.append(Guitar(parts[0], int(parts[1]), float(parts[2])))
```